src/use/use.py

```diff
@@ -86,39 +89,52 @@
 from importlib import metadata
 from logging import DEBUG, StreamHandler, getLogger, root
 from pathlib import Path
-from types import ModuleType
-from typing import Callable, Dict, List, Optional, Tuple, Union
+from inspect import Traceback
+from types import FrameType, ModuleType
+from typing import *
 from warnings import warn
 
-import toml
-
-try:
-    from pip._internal.utils.compatibility_tags import get_supported
-except ImportError:
-    get_supported = lambda: []
-
-import mmh3
+import mmh3 # type: ignore
```

Comment:
why's that?

Reply:
MyPy doesnt have type stubs for it that I could see